src/msg/swap/index.js

```diff
@@ -0,0 +1,58 @@
+function newMsgDeposit(despositor, tokenA, tokenB, slippage, deadline) {
+  return {
+    type: 'swap/MsgDeposit'.
+    value: {
+      depositor: depositor,
+      token_a: tokenA,
+      token_b: tokenB,
+      slippage: slippage,
+      deadline: deadline
+    }
+  }
+}
+
+function newMsgWithdraw(from, shares, minTokenA, minTokenB, deadline) {
+  return {
+    type: 'swap/MsgWithdraw',
+    value: {
+      from: from,
+      shares: shares,
+      min_token_a: minTokenA,
+      min_token_b: minTokenB,
+      deadline: deadline
+    }
+  }
+}
+
+function newMsgSwapExactForTokens(requester, extactTokenA, tokenB, slippage, deadline) {
+  return {
+    type: 'swap/MsgSwapExactForTokens',
+    value: {
+      requester: requester,
+      extact_token_a: extactTokenA,
```

Comment:
`extactTokenA ` should be `exactTokenA`

Reply:
`extact_token_a` -> `exact_token_a` as well